python/lsst/meas/base/forcedPhotCcd.py

```diff
@@ -143,7 +143,7 @@ class ForcedPhotCcdConfig(ForcedPhotImageConfig):
     doApplyUberCal = lsst.pex.config.Field(
         dtype=bool,
         doc="Apply meas_mosaic ubercal results to input calexps?",
-        default=False
+        default=True
```

Comment:
I agree with all of the above (including John's suggestion to rename this option, but I think that bit is out of scope for this issue).